Anlab.Mvc/Views/Lab/Confirmation.cshtml

```diff
@@ -2,7 +2,8 @@
 @model AnlabMvc.Models.Order.OrderReviewModel
 
 @{
-    ViewData["Title"] = "Confirmation";
+    ViewData["Title"] = "Confirmation";
+    ViewBag.OrderId = Model.Order.Id;
```

Comment:
@jSylvestre why is this needed?  same thing w/ a few other files too

Reply:
@srkirkland The ViewBag.OrderId tells the layout page to show the </> for the OrderDetails json call.